server/plugin.go

```diff
@@ -98,3 +105,16 @@ func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*mode
 
 	return post, ""
 }
+
+func contains(channel string, team string, list []string) bool {
+	for _, channelTeam := range list {
+		channelTeamSplit := strings.Split(channelTeam, "/")
+		if len(channelTeamSplit) == 2 {
```

Comment:
Maybe log an error if `len(channelTeamSplit) != 2`

Reply:
logging for != 1 & != 2.  (1 being the team only example you provided earlier)